nginx/dev/adage-nginx.conf

```diff
@@ -0,0 +1,63 @@
+server {
+    # default server - drop requests without a Host header
+    listen 80 default_server;
+    server_name "";
+    return 444;
+}
```

Comment:
This section is useful for security on a production server, but quite inconvenient for development. I think we can remove it and use `server_name "";` at what is currently line 13 below.

Note: I even hacked together a little `sed` command that edits our default `adage-nginx.conf` this way for a development setup. See the `tweak_nginx_for_dev` task in `adage-deploy/fabfile.py`.